index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+    crossorigin="anonymous"></script>
+  <title>Daa Nkwa</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+<body>
+  <header>
+    <!-- first navigation -->
+    <nav class="first-header bg-secondary">
+      <ul class="first-header-ul">
+        <li><a><img src="Img\facebook-letter-logo-svgrepo-com.svg" alt="Facebook"></a></li>
+        <li><a><img src="Img\twitter-white.svg" alt="twitter"></a></li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+    </nav>
```

Comment:
- Great Job for implementing previous reviewer suggestions, I think you forget to update the background color of the top headline, so please update to have a better UI 👍 
Figma Design 👇
![Screenshot 2023-06-30 001945](https://github.com/obibaadoma/captone_DaaNkwa/assets/52010789/41a4430f-eb30-4372-8b37-995d10c59b0e)
Your Design 👇
![Screenshot 2023-06-30 001816](https://github.com/obibaadoma/captone_DaaNkwa/assets/52010789/95bccc0f-5f8f-4545-b5ea-2ce84d33f73f)